source/content/debian-php-appserver.md

```diff
@@ -0,0 +1,144 @@
+
+
+## Known Changes and Requirements
+
+### OS Packages
+
+| Package | CentOS | Debian |
+|---------|---------|---------|
+| **OIC (Oracle Instant Client)** | Available for PHP 7.4-8.3 | Available for PHP 8.2-8.4 |
+| **IBM PHP ODBC Driver** | Available for PHP 7.4-8.2 | Not available |
+| **wkhtmltopdf** | 0.12.5 available | Not available |
+| **Apache Tika** | 1.18, v1.21 available | Not available <sup>1</sup> |
+| **PhantomJS** | 1.7.0, 2.1.1 available | Not available |
+
+<sup>1</sup> For <code>search_api_attachments</code> users, we recommend using the Solr Extractor built into the Pantheon Search Solr server, rather than using Tika on the appserver. This is configured at <code>/admin/config/search/search_api_attachments</code>. 
+
+### PHP Extensions
+
+| Extension | CentOS | Debian |
+|-----------|---------|---------|
+| **oci8** | Available for PHP 7.4-8.3<br/>3.3.x | Available for PHP 8.2-8.4<br/>3.4.x |
+| **ioncube** | Available for PHP 7.1 only | Not available |
+| **pdo_sqlsrv** | Available for PHP 7.2 only<br/>5.2.0 | Not available |
+| **curl** | v7.61.1 | 7.88.1 |
+| **gd** | 2.1.0 | 2.3.3 |
+| **iconv** | 2.28 | 2.36 |
+| **mongodb** | 1.14 | 1.20 |
+| **openssl** | 1.1.1k | 3.0.15 |
+| **pdo_pgsql** | 13.5 | 15.12 |
+| **pdo_sqlite** | 3.26.0 | 3.40.1 |
+| **pgsql** | 13.5 | 15.12 |
+| **redis** | 5.3.7<br/>Compression types: zstd | 6.1<br/>Compression types: zstd, lzf, lz4 |
+| **sqlite3** | 3.26.0 | 3.40.1 |
+
+Does your application require an OS package or PHP extension that is no longer available? Please reach out to us to discuss compatibility by [filling this form](https://docs.google.com/forms/d/e/1FAIpQLSfbgXOPRq4ylGgtGLHsQ68dwc_11eUzO7Y2X6PjIB3oy9rUxA/viewform?usp=header) or contacting your Customer Success Manager.
+
+### Networking
+
+| Protocol | CentOS | Debian |
+|---------|---------|---------|
+| **TLS** | 1.0-1.3 | 1.2-1.3 <sup>1</sup>|
+
+<sup>1</sup> Review the <a href="/tls-compatibility">TLS documentation</a> to check which version of TLS your external connections are using, and how to ensure you are compatible with this change.
+
+### CMS Version Requirements
+
+To provide the most secure environment, you may need to upgrade your CMS version to switch to the Debain-based appserver. If your CMS + Version is not listed below, it can be considered compatible.
+
+| CMS | Version | Minimum Required Version for Debian Appserver |
+|---------|---------|---------|
+| Drupal | 7.x | 7.58+ |
+| Drupal | 8.0.x | 8.4.6+ |
+| Drupal | 8.1.x | 8.4.6+ |
+| Drupal | 8.2.x | 8.4.6+ |
+| Drupal | 8.3.x | 8.4.6+ |
+| Drupal | 8.4.x | 8.4.6+ |
+| Drupal | 8.5.x | 8.5.1+ |
+
+
+<sup>1</sup> Review the [TLS documentation](/tls-compatibility) to check which version of TLS your external connections are using, and how to ensure you are compatible with this change.
+
+## Known Issues
+
+- SFTP mode is currently non-functional. Git mode is available.
```

Comment:
Just noting that this is a significant blocker. Is there an estimated date/timeline for when SFTP will be functional?